test/verify/check-system-info

```diff
@@ -827,7 +827,12 @@ password=foobar
         self.assertLess(initial_usage, 80)
         # allocate an extra 300 MB; this may cause other stuff to get unmapped,
         # thus not exact addition, but usage should go up
-        mem_hog = m.spawn("MEMBLOB=$(yes | dd bs=1M count=300 iflag=fullblock); touch /tmp/hogged; sleep infinity", "mem_hog.log")
+        #
+        # The "true" after "sleep" is there to prevent bash from
+        # replacing it's own process with the sleep (as a "tail call
+        # optimization") and thereby dropping the memory blob too early.
+        #
+        mem_hog = m.spawn("MEMBLOB=$(yes | dd bs=1M count=300 iflag=fullblock); touch /tmp/hogged; sleep infinity; true", "mem_hog.log")
```

Comment:
TIL! Thanks for figuring this out. In another place I'm invoking awk as I couldn't get bash to work reliably. Now I know why!